hardware-handler/nightmare_movement_engine/src/main.py

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env python3
 
-import time
+import os
+import sys
 import numpy as np
 
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))  # noqa
```

Comment:
,,,

Reply:
not solveable since workdir is managed by roslaunch